coniql/caplugin.py

```diff
@@ -127,6 +128,13 @@ def channel_from_update(
                 )
                 self.cached_status = status
 
+        if connected is not None:
```

Comment:
Why not just use time_value.ok instead of passing connected?

Reply:
Mostly just a relic from an earlier version. You could say it's more explicit though: it means we are tightly coupled to the monitor that was created outside of the class. True already though.

Reply:
OK, I have changed this in the latest commit.